doc/source/whatsnew/v1.1.0.rst

```diff
@@ -229,6 +229,8 @@ Performance improvements
   sparse values from ``scipy.sparse`` matrices using the
   :meth:`DataFrame.sparse.from_spmatrix` constructor (:issue:`32821`,
   :issue:`32825`,  :issue:`32826`, :issue:`32856`, :issue:`32858`).
+- Performance improvement in :meth:`Series.sum` for nullable (integer and boolean) dtypes (:issue:`30982`).
```

Comment:
is `DataFrame.sum` affected?

Reply:
No, since we don't use the EA implementation for reductions in DataFrame reductions, that's my other PR